recipes/msfragger/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# These values are submitted to the MSFragger site when downloading the application zip.
+if [[ -z "$NAME" ]]; then
+    NAME="${USERNAME:-bioconda}";
+fi
+if [[ -z "$EMAIL" ]]; then
+    EMAIL="${NAME}@${HOSTNAME:-bioconda.org}";
+fi
+if [[ -z "$INSTITUTION" ]]; then
+    INSTITUTION="${HOSTNAME:-bioconda.org}";
+fi
+
+# Create directories
+mkdir -p "$PREFIX/tools"
+
+# Add wrapper python script and link to unprefixed name.
+cp "$RECIPE_DIR/msfragger.py" "$PREFIX/bin"
+ln -s "$PREFIX/bin/msfragger.py" "$PREFIX/bin/msfragger"
```

Comment:
I'd probably put `msfragger.py` together with the JAR files, and have only the symlink in `$PREFIX/bin` .